sklearn/linear_model/tests/test_quantile.py

```diff
@@ -120,7 +126,7 @@ def test_quantile_estimates_calibration(q):
     quant = QuantileRegressor(
         quantile=q,
         alpha=0,
-        solver_options={"lstsq": False},
+        solver="highs",
```

Comment:
This tests breaks on old scipy versions (<1.6.0) that we still support for now.

Reply:
There are other tests to be updated similarly. We could probably have a help to chose the solver to use in the test based on the installed version of scipy.

Reply:
I did a small fixture to get the default value to use depending of the version.